ui/src/App.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { MouseEvent } from "react";
+import moment from "moment";
+import { useState } from "react";
+import { Calendar, momentLocalizer } from "react-big-calendar";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+import { CalendarEvent } from "./configs/CalendarEvent";
+import s from "./App.module.css";
+
+const currentHour = new Date().getHours();
+const currentMinute = new Date().getMinutes();
+const padNumberWith0 = (num: Number) => num.toString().padStart(2, "0");
+const defaultStartTime = `${padNumberWith0(currentHour)}:${padNumberWith0(currentMinute)}`;
+const defaultEndTime = `${padNumberWith0(currentHour + 1)}:${padNumberWith0(
+  currentMinute
+)}`;
+
+const localizer = momentLocalizer(moment);
+
+const formatDate = (date: Date) => {
+  return [
+    date.getFullYear(),
+    padNumberWith0(date.getMonth() + 1),
+    padNumberWith0(date.getDate()),
+  ].join("-");
+}
```

Comment:
All of this code could have types added to it..

e.g. const defaultEndTime: <type> = ...

Look into adding types for return types as well, e.g. if a function returns a string, etc.